clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/GCEUtil.groovy

```diff
@@ -188,6 +188,28 @@ class GCEUtil {
     }
   }
 
+  static boolean isShieldedVmCompatible(Image image) {
+    java.util.List<GuestOsFeature> guestOsFeatureList = image.getGuestOsFeatures()
+    if (guestOsFeatureList == null || guestOsFeatureList.size() == 0) {
+      return false
+    }
+
+    boolean isUefiCompatible = false;
+    boolean isSecureBootCompatible = false;
+    for (def feature : guestOsFeatureList) {
+      if (feature.getType() == "UEFI_COMPATIBLE") {
+        isUefiCompatible= true
+        continue
```

Comment:
Are there any guest features that are compatible with both UEFI and secure boot?

Reply:
According the Google's documentation both features must be available to recognize the image as a shielded image.

**Once the disk is ready, the next step is to create an official GCP image using the gcloud
command. In particular, you must specify UEFI_COMPATIBLE and SECURE_BOOT in the
--guest-os-features flag. These values allow GCE to recognize the image as a shielded
image. For**

Reply:
:+1: